src/tests/ftest/util/soak_utils.py

```diff
@@ -56,6 +60,27 @@ def ddhhmmss_format(seconds):
             "%H:%M:%S", time.gmtime(seconds % 86400)))
 
 
+def get_id():
+    """Increment a counter to generate job ids
+
+    Returns:
+        int : next counter value
+    """
+    return next(id_counter)
+
+
+def debug_logging(log, enable_debug_msg, log_msg):
```

Comment:
This may be better defined as a `SoakTestBase` method, where calls would look like `self.debug_logging("some message")` or `test.debug_logging("some message")`.